### PR TITLE
[fix] publish

### PR DESCRIPTION
**Pull request by ds300**
- fixes an invalid usage of process.env in the editor package
- fixes some bublic paths in the publishing infra code

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-yarn lazy run build-api
+npx lazy run build-api
```
Comment (SomeHats) [inline .husky/pre-commit:4]: is this to get around these not triggering correctly from within `bublic` due to a missing `YARN_RC_FILENAME`?

lu and i did an alternative version of this change where we checked if the containing folder was named bublic and manually set that var appropriately

**Review by SomeHats — APPROVED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (SomeHats) [reply .husky/pre-commit:4]: (this is fine too but if we wanted to stick with yarn for consistency then that works too!)

Comment (SomeHats) [reply .husky/pre-commit:4]: https://github.com/tldraw/tldraw/pull/1215/files#diff-d2bc4bbf14eadc292d84e0ef5f7a93115c23557f533809fc80b896934291529dR4-R8

Comment (ds300) [reply .husky/pre-commit:4]: > is this to get around these not triggering correctly from within bublic due to a missing YARN_RC_FILENAME?

Yeah exactly. npx does the right thing and shaves 30ms or so off the start time, so I'm inclined to stick with it.

**Pull request merged**